app/test_main.py

```diff
@@ -1,3 +1,70 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetHumanAge:
+    @pytest.mark.parametrize(
+        "init_cat_age,init_dog_age,expected_list_of_age",
+        [
+            pytest.param(
+                0,
+                0,
+                [0, 0],
+                id="should return zeros list if the initial values are 0"
+            ),
+            pytest.param(
+                14,
+                14,
+                [0, 0],
+                id="should return zeros list if the initial values are < 15"
+            ),
+            pytest.param(
+                15,
+                15,
+                [1, 1],
+                id="should return 1 year list if the initial "
+                   "values are more than 15 and less than 24"
+            ),
+            pytest.param(
+                23,
+                23,
+                [1, 1],
+                id="should return 1 year list if the initial "
+                   "values are  less than 24"
+            ),
+            pytest.param(
+                24,
+                24,
+                [2, 2],
+                id="should return 2 year list if the initial values are 24"
+            ),
+            pytest.param(
+                27,
+                27,
+                [2, 2],
+                id="should return 2 year list if the initial values are 27, 27"
+            ),
+            pytest.param(
+                28,
+                28,
+                [3, 2],
+                id="should return 3 for cat and 2 years for dog if "
+                   "the initial values are 28, 28"
```

Comment:
The expected result `[3, 2]` for initial ages of 28 for both cat and dog might not be correct depending on the logic of `get_human_age`. Ensure that this expectation aligns with the function's logic for converting pet ages to human years.